packages/tofu/src/inspectSesCompat.js

```diff
@@ -37,9 +66,15 @@ function inspectSesCompat(ast) {
     }
   })
   // check for mutations to named intrinsics
-  const sesNamedIntrinsics = Reflect.ownKeys(sesAllowlist).filter(
-    (k) => k in global && typeof sesAllowlist[k] === 'object'
-  )
+  const sesNamedIntrinsics = Reflect.ownKeys(sesAllowlist)
+    .filter(
+      (k) =>
+        k in global &&
+        // @ts-expect-error - filter loses specificity
+        typeof sesAllowlist[k] === 'object'
+    )
+    .map(String)
```

Comment:
Can we force-cast instead of looping again?

Reply:
I could make it a `reduce`, but `Reflect.ownKeys()` returns _all_ non-private keys in an object, so for our purposes, we will want to cast to `string`.

Reply:
[edit - I deleted what I initially wrote]

I'll need to take a second look here, because maybe we do want to allow Symbols too, not just string keys, even if the current implementation was not ready for that.